packager/certcache/certcache.go

```diff
@@ -429,9 +429,9 @@ func (this *CertCache) readOCSP(allowRetries bool) ([]byte, time.Time, error) {
 
 // Print # of retries, wait for specified time and returned updated wait time.
 func waitForSpecifiedTime(waitTimeInMinutes int, numRetries int) int {
-	log.Printf("Retrying OCSP server: retry #%d\n", numRetries)
+	log.Printf("Retrying OCSP server: retry #%d", numRetries)
```

Comment:
Did you remove the \n on purpose? Did you mean to change it to Println then?

Reply:
Yeah, the \n is redundant. I saw this done on webpkg. It's a difference between `fmt` and `log`: https://golang.org/pkg/log/#pkg-overview:~:text=Every%20log%20message%20is%20output%20on,newline%2C%20the%20logger%20will%20add%20one.